oneview_redfish_toolkit/blueprints/computer_system.py

```diff
@@ -222,6 +222,7 @@ def create_composed_system():
 
         server_profile = ComputerSystem.build_server_profile(
             body["Name"],
+            body.get("Description"),
```

Comment:
the get function is able to handle if the "Description" key does not exists, it will return None, for the cases that the user doesn't define any Description for the composed system.